packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -462,24 +482,83 @@ export function TransferPanelMain({
       setQueryParams({ l2ChainId })
     }
 
+    function modifyOptions(selectedChainId: ChainId, direction: 'from' | 'to') {
+      // Add L1 network to the list
+      return [l1.network, ...options]
+        .filter(option => {
+          // Remove selected network from the list
+          return option.chainID !== selectedChainId
+        })
+        .map(option => {
+          // Set disabled options (currently One<>Nova is disabled)
+          return {
+            ...option,
+            disabled:
+              direction === 'from'
+                ? (to.chainID === ChainId.ArbitrumNova &&
+                    option.chainID === ChainId.ArbitrumOne) ||
+                  (to.chainID === ChainId.ArbitrumOne &&
+                    option.chainID === ChainId.ArbitrumNova)
+                : (from.chainID === ChainId.ArbitrumNova &&
+                    option.chainID === ChainId.ArbitrumOne) ||
+                  (from.chainID === ChainId.ArbitrumOne &&
+                    option.chainID === ChainId.ArbitrumNova),
```

Comment:
Not a big fan of this here but it's also gonna get removed soon anyway so let's keep it as is.